src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadImagesData.php

```diff
@@ -31,13 +32,21 @@ class LoadImagesData extends DataFixture
     public function load(ObjectManager $manager)
     {
         $finder = new Finder();
-        $uploader = $this->get('sylius.image_uploader');
+
+        $dm = $this->get('doctrine_phpcr.odm.document_manager');
+        $uploadImageHelper = $this->get('cmf_media.persistence.phpcr.upload_image_helper');
 
         foreach ($finder->files()->in(__DIR__.$this->path) as $img) {
-            $image = $this->getProductVariantImageRepository()->createNew();
-            $image->setFile(new UploadedFile($img->getRealPath(), $img->getFilename()));
-            $uploader->upload($image);
 
+            // Upload the Image document in CMF Media repository
+            $file = new UploadedFile($img->getRealPath(), $img->getFilename(), null, null, null, true);
+            $media = $uploadImageHelper->handleUploadedFile($file, 'Symfony\Cmf\Bundle\MediaBundle\Doctrine\Phpcr\Image');
+            $dm->persist($media);
+            $dm->flush();
+
+            // Create Image entity (which is holding a reference to above the Image document)
```

Comment:
Is this comment really needed? I think it's pretty clear, what you're doing here ;)